Library/Formula/llvm.rb

```diff
@@ -158,9 +161,16 @@ def install
         # Let's use clang to build libcxx. It fails with gcc.
         ENV["CC"] = bin/"clang"
         ENV["CXX"] = bin/"clang++"
+        mktemp do
+          system "cmake", "-G", "Unix Makefiles", "#{buildpath}/projects/libcxx", *std_cmake_args
```

Comment:
Oh, I see. You copied it [from above](https://github.com/DoomHammer/linuxbrew/blob/llvm-for-linux-fixes/Library/Formula/llvm.rb#L154). We'll just go with that, then.